presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchClient.java

```diff
@@ -454,4 +464,51 @@ public int compare(String left, String right)
             return rightLength - leftLength;
         }
     }
+
+    static TransportClient createTransportClient(ElasticsearchConnectorConfig config, TransportAddress address)
+    {
+        return createTransportClient(config, address, Optional.empty());
+    }
+
+    static TransportClient createTransportClient(ElasticsearchConnectorConfig config, TransportAddress address, Optional<String> clusterName)
+    {
+        Settings settings;
+        Builder builder;
+        TransportClient client;
+        if (clusterName.isPresent()) {
+            builder = Settings.builder()
+                    .put("cluster.name", clusterName.get());
+        }
+        else {
+            builder = Settings.builder()
+                    .put("client.transport.ignore_cluster_name", true);
+        }
+        switch (config.getCertificateFormat()) {
```

Comment:
Maybe use `io.airlift.security.pem.PemReader#isPem(java.lang.String)` instead of making users deal with it.